README.md

```diff
@@ -37,7 +37,7 @@ commonly used with React components.
 Here is an example of using PropTypes with a React component, which also
 documents the different validators provided:
 
-```javascript
+```jsx
 import React from 'react';
```

Comment:
Looks like we don't actually use jsx in this code block, but in case we did in the future I think this change still makes sense.